boards/shields/seeed_w5500/doc/index.rst

```diff
@@ -0,0 +1,66 @@
+.. _seeed_w5500:
+
+Seeed W5500 Ethernet Shield
+###########################
+
+Overview
+********
+
+Seeed `W5500 Ethernet Shield`_ is Arduino connector shield with `W5500`_ 10/100
+MBPS stand alone Ethernet controller with on-board MAC & PHY, 16 KiloBytes for
+FIFO buffer and SPI serial interface. This shield additionally has Grove UART
+connector, Grove I2C connector and SD card slot.
+
```

Comment:
Seeed `W5500 Ethernet Shield`_ is an Arduino connector shield with:
- `W5500`_ 10/100 MBPS stand alone Ethernet controller with on-board
MAC & PHY and 16 KiloBytes for FIFO buffer
- SPI serial interface
- Grove UART connector
- Grove I2C  connector
- SD card slot.

Reply:
Applied, thanks!